packages/viewer/src/scenes/raycaster.ts

```diff
@@ -13,11 +18,20 @@ export class Raycaster {
    * the given point.
    *
    * @param point The point to cast from looking for intersections.
+   * @param include (optional) Additional fields to be returned with each hit item.
+   * See the `HitResultInclude` type for available fields.
    */
   public async hitItems(
-    point: Point.Point
+    point: Point.Point,
+    include?: HitResultInclude[]
   ): Promise<vertexvis.protobuf.stream.IHitItemsResult | undefined> {
-    const res = await this.stream.hitItems({ point }, true);
+    const res = await this.stream.hitItems(
+      {
+        point,
+        includeBoundingBox: include && include.includes('bounding-box'),
```

Comment:
could an interface to FSS here have been to pass the array of 'includes' to fetch. then this logic to indicate specifically the bounding box should be included could exist in FSS? are there advantages to making the payload to FSS some number of keys on what to include?

I'm not confident either way on this, just a question on the decision.

Reply:
You might be able to use protobuf enums to make this typesafe.

Reply:
Yeah, we certainly could still go that route if we wanted to change this. Currently the payload is mainly structured this way to follow similar structuring to the protos we have for the scene-service protos. I'd certainly be open to changing that though if FSS seems like the better place to do the mapping to those scene-service protos